basic/ani.txt

```diff
@@ -1,9 +1,10 @@
-MODE7
+MODE7:PRINT:*FX21,0
 HIMEM=HIMEM-&200
-PROCmenu
+PROCmenu:CLS
```

Comment:
Clear screen so you don't see residual old image data when prog next switches back to MODE7.